components/cards/style.css

```diff
@@ -0,0 +1,241 @@
+.card__content {
+  border: 1px solid var(--gray-light-secondary);
+  padding: 2rem;
+  margin-top: 2rem;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.3rem;
+  box-shadow: 5rem 5rem 5rem 5rem;
+  box-shadow: 0px 0px 5px 0px var(--gray-light-primary);
```

Comment:
Why two same properties in the same class?

Reply:
will remove it